aegir/service.sls

```diff
@@ -0,0 +1,10 @@
+{% from "aegir/map.jinja" import aegir with context %}
+
+aegir_hostmaster:
+  service.running:
+    - name: {{ aegir.hostmaster.service }}
+    - enable: true
+    - require:
+      - pkg: aegir
```

Comment:
How will require be satisfied?  Are you missing include?

Reply:
You're right. Added.

Reply:
Did you push it?

Reply:
Yes, [line 3](https://github.com/saltstack-formulas/aegir-formula/pull/4/files#diff-34f2299e5a32f710c65673af57bac1c2R3) above :smile:

Reply:
Sorry, maybe I never refreshed the page.